requirements.txt

```diff
@@ -1,5 +1,6 @@
 --extra-index-url https://download.pytorch.org/whl/cpu
 click
+dredge
```

Comment:
I'm a bit confused here -- we don't have dredge on pypi

Reply:
Oups: 

https://pypi.org/project/dredge/

We had the same issue with one of our modules !